### DISCUSSION TITLE
Access tokens don't work with https://github.com/... URLs. Should they?

### DISCUSSION THREAD

**Discussion by iliakan** [community/community#49543] [Other Feature Feedback, Questions, & Ideas]
I created a personal access token (classic).

It works for API, but not for direct github access.

This kind of request works:
```
curl -H "Authorization: Token [...]" https://api.github.com/user/issues
```

...But this one doesn't work:
```
curl -H  "Authorization: Token [...]" 'https://github.com/iliakan/empty-private/info/refs?service=git-upload-pack'
```

Is it a bug or a feature?

I'm trying some low-level stuff, so I need the "direct" (latter) type of access, not API.
 
If it's a feature, is there any way to pass an oauth token to such requests?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @iliakan ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
